lrpar/src/lib/ctbuilder.rs

```diff
@@ -197,9 +197,9 @@ where
         }
     }
 
-    /// Set the input grammar path to a file in this project's `src` directory. This will also set
-    /// the output path (i.e. you do not need to call [CTParserBuilder::output_path]) and module
-    /// name (i.e. you do not need to call [CTParserBuilder::mod_name]).
+    /// Set the input grammar path to a file relative to this project's `src` directory. This will
```

Comment:
This is what I was referring to in my earlier comment. The doc string suggests this is always relative to the project root, but that's not true if the binary is run from somewhere else, e.g. `/home/lukas/grmtools/bin/grmtools grm.y`.

Reply:
There is no `grmtools` binary so I'm not sure this can really happen.

Reply:
This was just an example. There will be some binary that the user runs, and since they can run it from anywhere, I feel like claiming this is always relative to the project root is misleading. Unless you mean it's only possible to run this from within the project's root due to other dependencies?

Reply:
I'm sure someone can work out how to run it in a weird way, but it's only documented as being runnable from a project's `build.rs` and I think there that CWD == project root (but I might be wrong).

Reply:
Ah, it depends on `build.rs` which takes the grammar path and the compiles the parser. In that case I agree that it's unlikely someone will run it from anywhere but the project root.